bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -17,17 +17,29 @@
 
 import java.util.concurrent.TimeUnit;
 
-import com.google.cloud.bigtable.grpc.BigtableGrpcClient;
-
 /**
  * Options for retrying requests, including back off configuration.
  */
 public class RetryOptions {
 
-  public static int DEFAULT_STREAMING_BUFFER_SIZE = BigtableGrpcClient.SCANNER_BUFFER_SIZE;
-  // We can timeout when reading large cells with a low value here. With a 10MB
-  // cell limit, 60 seconds allows our connection to drop to ~170kbyte/s. A 10 second
-  // timeout requires 1Mbyte/s
+  public static int DEFAULT_STREAMING_BUFFER_SIZE = 32;
+
+  /**
+   * Flag indicating whether or not grpc retries should be enabled.
+   * The default is to enable retries on failed idempotent operations.
+   */
+  public static boolean ENABLE_GRPC_RETRIES_DEFAULT = true;
+
+  /**
+   * Flag indicating whether or not to retry grpc call on deadline exceeded.
+   * This flag is used only when grpc retries is enabled.
+   */
+  public static boolean ENABLE_GRPC_RETRY_DEADLINEEXCEEDED_DEFAULT = true;
+
+  /** We can timeout when reading large cells with a low value here. With a 10MB
+   * cell limit, 60 seconds allows our connection to drop to ~170kbyte/s. A 10 second
+   * timeout requires 1Mbyte/s
+   */
   public static int DEFAULT_READ_PARTIAL_ROW_TIMEOUT_MS =
       (int) TimeUnit.MILLISECONDS.convert(60, TimeUnit.SECONDS);
```

Comment:
Can't remove the casting here.  .convert() returns a long, and there's no auto casting from long to int.